docker/nextjs.monorepo.optimized.Dockerfile

```
@@ -0,0 +1,109 @@
+
+ARG APP_NAME=app
+
+ENV NODE_ENV=production
+ENV NEXT_TELEMETRY_DISABLED=1
+
+# Create non-root user
+RUN addgroup --system --gid 1001 nodejs && \
+    adduser --system --uid 1001 nextjs
+
+# Copy public assets (may not exist for all apps)
+COPY --from=builder --chown=nextjs:nodejs /app/apps/${APP_NAME}/public ./public 2>/dev/null || true
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)